src/components/Menu/MenuGridItem/MenuGridItem.jsx

```diff
@@ -1,12 +1,12 @@
-import React, { useRef, forwardRef, useCallback } from "react";
-import PropTypes from "prop-types";
 import cx from "classnames";
+import React, { forwardRef, useCallback, useRef } from "react";
+import PropTypes from "prop-types";
 import useMergeRefs from "../../../hooks/useMergeRefs";
-import "./MenuGridItem.scss";
```

Comment:
no css modules file?

Reply:
Stylesheet contained only this:
`@import "../../../styles/themes";`
So, I've removed it, but left the original classnames to keep existing overrides